tilings/strategy_pack.py

```diff
@@ -56,6 +64,43 @@ def make_fusion(
             pack = pack.add_initial(strat.FusionFactory(tracked=tracked), "fusion")
         return pack
 
+    def make_interleaving(self, tracked=True) -> "TileScopePack":
+        """
+        Return a new pack where the factor strategy is replaced with an
+        interleaving factor strategy.
+        """
+
+        def replace_list(strats):
+            """Return a new list with the replaced 1x1 strat."""
```

Comment:
I think the docstring is wrong, smell like copy-pasting ;)

Reply:
Yes, a copy paste error :(

Reply:
fixed